sodium/test/crypto_box_test.pony

```diff
@@ -38,7 +38,25 @@ class CryptoBoxTest is UnitTest
     try CryptoBox.open(crypt', nonce', csk, bpk)
       h.fail("Cyril shouldn't be able to open Bob's message to Alice.")
     end
-    
+
+    ///
+    // Key pairs generated with seeds
+
+    (let sks, let pks) = CryptoBox.seed_keypair(CryptoBoxSeed("Hello seeds!"))
```

Comment:
You said that the seed size is 32 bytes, but this is 12.  Shouldn't this call `error` out?